src/main/java/com/ibm/eventstreams/connect/jdbcsink/database/builder/CommandBuilder.java

```diff
@@ -0,0 +1,172 @@
+package com.ibm.eventstreams.connect.jdbcsink.database.builder;
```

Comment:
Copy right headers are required at the top of new files and edited files will need updating